game/src/MusicConductor/SongInfo.gd

```diff
@@ -0,0 +1,11 @@
+extends Node
+class_name SongInfo
+
+var fullyQualifiedPath : String = ""
+var readableName : String = ""
+var loadedStream : Resource
+
+func _init(dirname:String, fname:String):
+		fullyQualifiedPath = dirname + fname
+		readableName = fname.replace(".mp3","").replace(".ogg","").replace(".wav","").replace("_", " ")
```

Comment:
Thinking about it, probably should prefer
```suggestion
		readableName = fname.get_basename()
```

Reply:
This would be a lot cleaner, as long as you can exclude godot's .import files, again, a change that we don't need right now, but would be nice.

Reply:
Changed in most recent commit